security-analysis/security-analysis-api/src/test/resources/ActionFileTestV1.0.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version" : "1.0",
```

Comment:
Why did you put this file in 1.1? It's now exactly the same file as ActionFileTest.json

Reply:
@miovd you cannot change "value" to "tapPosition" because it is not always a tap position. When relative, it is a delta. Value has been chosen in consequence.  I don't understand why we have this change...

Reply:
Indeed, my mistake...

Reply:
You should stay in 1.0.